src/tools/schema2proto.ts

```diff
@@ -39,6 +39,7 @@ Examples [Must be run from Arcs repository root]
 }
 
 // Converts schema definitions specifies in Manifest files to proto2 specifications.
+// tslint:disable-next-line: no-floating-promises
```

Comment:
so... shall we do this, or `void` them? I've been voiding and recommending voiding.

voiding is not beautiful, but at least (I presume) will work file with bazel if it comes to that. WDYT?

Reply:
I hope it would become an idiom that voiding means "I know what I'm doing, hold my bear".

Reply:
This seemed more explicit to me, but you're right about bazel. Switched to void.